test/scenarios/bookshop/read.test.js

```diff
@@ -53,6 +53,13 @@ describe('Bookshop - Read', () => {
     expect(columns).to.contain('title')
   })
 
+  test('Path expression with null value', async () => {
```

Comment:
1. the test name is incorrect
2. there is already a test doing this as was shared with you before [here](https://github.com/cap-js/cds-dbs/blob/9f8906ef1e916aaff3bdcf3faed22740e814b7dc/test/compliance/SELECT.test.js#L45)

Reply:
1. Please make a suggestion for the test name if you don't  like it.
2. This is an integration test (I suppose it should run with Hana, Postgres..), the other one is unit.